PWGHF/Tasks/taskD0parametrizedPID.cxx

```diff
@@ -131,8 +131,8 @@ struct TaskD0parametrizedPIDMC {
           maxFiducialY = -0.2 / 15 * ptGen * ptGen + 1.9 / 15 * ptGen + 0.5;
           minFiducialY = 0.2 / 15 * ptGen * ptGen - 1.9 / 15 * ptGen - 0.5;
         }
-        auto etaProng0 = std::abs(particle.daughter0_as<McParticlesHf>().eta());
```

Comment:
This didn't work?

Reply:
I mean `particle.daughter_as<McParticlesHf>().front().eta()`

Reply:
The previous comment was not correct: as discussed in person I needed to use `begin()` instead of `front()` so it works now